dissect/target/loaders/log.py

```diff
@@ -7,21 +7,22 @@
 
 class LogLoader(Loader):
     LOGS_DIRS = {
-        "EvtxPlugin": "sysvol/windows/system32/winevt/logs",
-        "EvtPlugin": "sysvol/windows/system32/config",
+        ".evtx": "sysvol/windows/system32/winevt/logs",
+        ".evt": "sysvol/windows/system32/config",
     }
 
     @staticmethod
     def detect(path: Path) -> bool:
         return False
 
-    def _map_entry(self, entry: Path) -> str:
-        logs_dir_key = entry.suffix[1:].capitalize()
-        return str(Path(self.LOGS_DIRS[f"{logs_dir_key}Plugin"]).joinpath(entry.name)).lower()
-
     def map(self, target: Target) -> None:
         self.target = target
         vfs = VirtualFilesystem()
-        for entry in self.path.parent.rglob(self.path.name):
-            vfs.map_file(str(self._map_entry(entry)), str(entry))
+        for entry in self.path.parent.glob(self.path.name):
+            ext = entry.suffix.lower()
+            mapping = self.LOGS_DIRS.get(ext, None)
+            if mapping is None:
+                continue
+            mapping = str(Path(mapping).joinpath(entry.name)).lower()
```

Comment:
Is the `lower()` necessary?

Reply:
Yes, sometimes those files are named .Evt :-(

Reply:
Would it not be better to make the `VirtualFilesystem` be `case_sensitive=False` in this case? Since we're (at least in the current scenario) exclusively emulating a Windows filesystem.

Reply:
Unfortunately that does not seem to work because the plugin internally uses glob() which even on a case insensitive fs will take casing into account. Overriding glob for case insensitive filesystems may work but may break things.

Reply:
That sounds like a bug in glob that should be fixed. It should be case-insensitive on case-insensitive filesystems and case-sensitive on case-sensitive filesystems.

Reply:
I don't think that is quite a trivial thing to do (https://stackoverflow.com/questions/8151300/ignore-case-in-glob-on-linux) and it might break stuff that accidentally depends on this behaviour. Doesn't this warrant a separate issue?

Reply:
But this is about the case-sensitiveness of `VirtualFilesystem` right? The `fs.glob()` should already be case-insensitive: https://github.com/fox-it/dissect.target/blob/main/dissect/target/helpers/fsutil.py#L919-L923 and the `TargetPath.glob()` should also already be case-insensitive: https://github.com/fox-it/dissect.target/blob/main/dissect/target/helpers/fsutil.py#L287

Reply:
Right, I see that this is a simple error in my code after all. Thanks for your guidance.